capstone/src/main/java/com/google/sps/data/Form.java

```diff
@@ -2,28 +2,28 @@
 import java.io.*;
 import com.google.appengine.api.datastore.Entity;
 
-
+/**
+ * The Form class holds the user's forms for their classes
+ */
 public class Form {
 
     private String editURL;
     private String URL;
-    private int id;
+    private String id;
     private boolean isPublished;
 
-    public Form(String editURL, String URL, int id) {
+    public Form(String editURL, String URL, String id) {
         this.editURL = editURL;
         this.URL = URL;
         this.id = id;
         this.isPublished = false;
     }
 
-    // Setters
-    public void publish(){
-        this.isPublished = true;
-    }
-
-    public void unpublished(){
-        this.isPublished = false;
+    public Form(Entity entity){
+        this.editURL = (String) entity.getProperty("editURL");
+        this.URL = (String) entity.getProperty("URL");
+        this.id = (String) entity.getProperty("id");
+        this.isPublished = Boolean.parseBoolean((String) entity.getProperty("isPublished"));
```

Comment:
Can you just cast this into Boolean? This doesnt seem like it should be part of the API

Reply:
Yeah, casting as a Boolean sounds more correct.